PWM.py

```diff
@@ -0,0 +1,18 @@
+#code works along side GPIO.py
+#the init in GPIO.py must be called before any of these functions are called
+
+import RPi.GPIO as GPIO
+
+class signal:
+    def setPin(signal, pin, frequency):
```

Comment:
looks like `setPin()` always need to be called first before calling any of the other methods, so you should do this in the constructor.

Reply:
How would I do that with undefined number of PWM pins (in python specifically)? In C++ I would've used `auto`. I'm assuming something like:
`def __init__(self, SOMETHING IN PYTHON):`

Reply:
to answer your question, you can do
```python
def __init__(self, *pins)
```
and pass any number of pins in, where `pins` will be an array. you can look up *args and **kwargs to learn more about this.

more importantly though, why do you want to accept a variable number of pins? based on line 10
```python
self.pwm = RPi_IO.PWM(pin, frequency)
```
it looks like one signal can only have one pin anyway, so the constructor only needs to take in one pin and frequency and just call `setPin()`.

Reply:
You are right, I just forgot for a second how I setup it up lol. I'll make the change